lib/workers/pr/changelog/release-notes.ts

```diff
@@ -207,7 +207,9 @@ export async function getReleaseNotesMdFileInner(
 export function getReleaseNotesMdFile(
   project: ChangeLogProject
 ): Promise<ChangeLogFile | null> {
-  const cacheKey = `getReleaseNotesMdFile-${project.repository}-${project.apiBaseUrl}`;
+  const cacheKey = `getReleaseNotesMdFile-${project.repository}${
```

Comment:
```suggestion
  const cacheKey = `getReleaseNotesMdFile@v2-${project.repository}${
```

Something like this will ensure existing cached entries are missed /ignored